services/argus/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.9.0 (2024-01-12)
+
+- **Feature**: Add `SampleLimit` to `UpdateScrapeConfigPayload`.
+  This field allows you to update the per-scrape limit on number of scraped samples that will be accepted. If more than this number of samples are present after metric relabeling the entire scrape will be treated as failed. The total limit depends on the service plan target limits \* samples.
```

Comment:
Not sure if you had an idea as to how to format this, but Markdown is just ignoring the newline

Reply:
Added newline